### ISSUE TITLE
Encoder_Loss is always NaN

### ISSUE THREAD

**Issue by Steven-N-Hart** [amandaberg/GANanomalyDetection#2]
(moving from closed PR)

I am looking at your Encoder loss, which is always producing NANs. I'm still digging through it with the MNIST and other datasets to find out why, but this is obviously causing the reconstructed images to not look correct. See for yourself:
```
loss = tf.reduce_mean(tf.abs(fake_images_out - fake_images_reconstructed))
loss = tfutil.autosummary('Encoder_Loss', loss)
```

**Comment by amandaberg** [comment]
Dear Steven,
I have never encountered this problem myself, neither with the MNIST dataset or others. From the two code lines that you included, it seems to me that the only reason for this to happen is if fake_images_out or fake_images_reconstructed contains NaNs.
